packages/documentation/src/stories/foundation/layout/Columns/columns.stories.ts

```diff
@@ -0,0 +1,255 @@
+import type { Args, Meta, StoryFn, StoryObj, StoryContext } from '@storybook/web-components';
+import { BADGE } from '../../../../../.storybook/constants';
+import { html } from 'lit';
+
+const meta: Meta = {
+  title: 'Foundations/Layout/Columns',
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+  },
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      <div class="p-regular column-example text-center">${story(args, context)}</div>
+    `,
+  ],
+  args: {
+    alignItems: 'align-items-start',
+    alignSelf: 'no self alignment',
+    justifyContent: 'justify-content-start',
+    offsetItem: 'offset-md-1',
+  },
+  argTypes: {
+    alignItems: {
+      name: 'Align Items',
+      description: 'Aligns the whole row vertically.',
+      control: {
+        type: 'select',
+      },
+      options: ['align-items-start', 'align-items-center', 'align-items-end'],
+      table: {
+        category: 'General',
+      },
+    },
+    alignSelf: {
+      name: 'Align Item 1',
+      description: 'Aligns the Item 1 vertically.',
+      control: {
+        type: 'select',
+      },
+      options: ['no self alignment', 'align-self-start', 'align-self-center', 'align-self-end'],
+      table: {
+        category: 'General',
+      },
+    },
+    justifyContent: {
+      name: 'Horizontal Alignement',
+      description: 'Aligns the Items horizontally.',
+      control: {
+        type: 'select',
+      },
+      options: [
+        'justify-content-start',
+        'justify-content-center',
+        'justify-content-end',
+        'justify-content-around',
+        'justify-content-between',
+        'justify-content-evenly',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+    offsetItem: {
+      name: 'Offset classes',
+      description: 'Increases the left margin of a column',
+      control: {
+        type: 'select',
+      },
+      options: [
+        'offset-md-1',
+        'offset-md-2',
+        'offset-md-3',
+        'offset-md-4',
+        'offset-md-5',
+        'offset-md-6',
+        'offset-md-7',
+        'offset-md-8',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const VerticalExample: Story = {
+  parameters: {
+    controls: { exclude: ['Horizontal Alignement', 'Offset classes'] },
+  },
+  render: (args: Args) => html`
+    <div class="container">
+      <div class="row-height row ${args.alignItems}">
+        <div class="col ${args.alignSelf === 'no self alignment' ? '' : args.alignSelf}">
```

Comment:
```suggestion
        <div class="col${args.alignSelf === 'no self alignment' ? '' : ` ${args.alignSelf`}">
```
Nitpick, but when there is no arg, there is a space at the end (see demo with default behavior).